facturacion_mexico/addendas/addenda_service.py

```
@@ -0,0 +1,193 @@
+		if not addenda_type:
+			return
+
+		try:
+			addenda_type_doc = frappe.get_cached_doc("Addenda Type", addenda_type)
+		except frappe.DoesNotExistError:
+			return
+
+		missing = []
+		for fd in addenda_type_doc.field_definitions or []:
+			if fd.is_mandatory and not addenda_values.get(fd.field_name):
+				missing.append(fd.field_label or fd.field_name)
```

Ruff (0.15.12)

[warning] Use a list comprehension to create a transformed list

(PERF401)